runtime/syntax/bend.yaml

```diff
@@ -0,0 +1,49 @@
+# Author: https://github.com/RohanVashisht1234
+filetype: bend
+
+detect:
+    filename: "\\.bend$"
+
+rules:
+      # keywords
+    - statement: "\\b(def|switch|case|return|if|else|when|match|λ|Some|data|let|use|object|fold|open|do|bind|Name|identity|Bool|ask|with)\\b"
+      # special method names
+    - symbol.operator: "\\b(bend|None|Nil|Result|type|lambda)\\b"
+      # definitions
+    - identifier: "def [a-zA-Z_0-9]+"
+      # keywords (declaration)
+    - statement: "\\b(Node|Leaf|Tree)\\b"
+      # constants (true, false, null, this)
+    - constant: "\\b(true|false|True|False)\\b"
+      # constants (types)
+    - type: "\\b(ud|str|int|float|obj|fib|bool|pat|type|any)\\b\\??"
+      # parentheses
+    - symbol.brackets: "([(){}]|\\[|\\])"
+      # extra operators
+    - symbol.operator: "(\\+|\\-|\\/|%|\\*|>>|<<|\\\\|\\^|&|~|!>)"
+      # Comparision operators
+    - symbol.operator: "(==|!=|<=?|>=?|\\bis\\b|\\?\\?)"
```

Comment:
I tried looking at documentation and examples a bit but is the `is` operator in Bend syntax?

Reply:
ok, I have removed `is`